services/121-service/src/user/user-emails/user-email-templates/user-email-templates.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@nestjs/common';
+
+import { EmailType } from '@121-service/src/user/user-emails/enum/email-type.enum';
+import { UserEmailTemplateInput } from '@121-service/src/user/user-emails/interfaces/user-email-template-input.interface';
+import { EmailTemplate } from '@121-service/src/user/user-emails/user-email-templates/interfaces/email-template.interface';
+import { emailTemplatePasswordReset } from '@121-service/src/user/user-emails/user-email-templates/templates/password-reset.template';
+import { emailTemplateRegistrationCreation } from '@121-service/src/user/user-emails/user-email-templates/templates/registration-creation.template';
+import { emailTemplateRegistrationCreationSSO } from '@121-service/src/user/user-emails/user-email-templates/templates/registration-creation-sso.template';
+
+@Injectable()
+export class UserEmailTemplatesService {
+  public buildEmailTemplate(
```

Comment:
Do we want to unit-test the actual outcome of this 'template juggling'?
Or would that be covered by e2e-tests?

The question would be "how complex do we want to make these tests"...

But currently it would still be possible to "break the end-product" of this whole "emails-services-chain"...